modelplugin/yang/ietf-interfaces@2014-05-08.yang

```diff
@@ -0,0 +1,111 @@
+module ietf-interfaces {
```

Comment:
these files are because they cannot be pulled from openconfig public repo ?

Reply:
Yes - I found it very difficult to get these particular versions of the YANG files - had to go back through the Git history in open configs to get these - they are a snapshot in time, and so I wanted to keep them versioned. In fact I want to rename ietf-interfaces because it has to be modified - only the typedefs are needed from it and I had to comment out the containers to get it to compile

Reply:
ok, it really shows that they want you to use the latest version! Interesting on the change/comment, why is that ?

Reply:
I changed ietf-interfaces@2014-05-08.yang to ietf-interfaces@2014-05-08.modified with a big comment at the top.
I haven't found which typedef exactly that the openconfig guys wanted from ietf-interfaces, but they redefine interfaces